src/Analysis/Ast/Impl/Modules/Resolution/MainModuleResolution.cs

```diff
@@ -190,6 +192,19 @@ public IPythonModule GetSpecializedModule(string name)
             foreach (var s in _userPaths) {
                 _log?.Log(TraceEventType.Information, $"    {s}");
             }
+
+            _log?.Log(TraceEventType.Information, "Aliased search paths:");
+            var zipRoot = CacheFolders.GetZipCacheRoot(_services.GetService<IOSPlatform>());
+            foreach (var s in zipPaths) {
+                // Alias ../test.zip => ../test folder
+                var aliasedPath = Path.Combine(zipRoot, Path.GetFileNameWithoutExtension(s.Path));
+                if(!Directory.Exists(aliasedPath)) {
+                    _fs.ExtractZip(s.Path, aliasedPath);
+                }
+                // User path has the .egg or .zip path, redirect it to have the new alias path
+                _userPaths = _userPaths.Add(aliasedPath);
```

Comment:
I add to userPaths, would there be a difference in adding to interpreter path if the zip came from there?

Reply:
Now that I think about it the aliased path is always under the analysis cache path, so should I add it to interpreter paths?